src/engine/MaxConstraint.cpp

```diff
@@ -345,16 +389,16 @@ List<PiecewiseLinearConstraint::Fix> MaxConstraint::getPossibleFixes() const
     //  3. if f is greater than all variables except 1
 
     if ( FloatUtils::gt( fValue, maxVal ) )
-	{
-	    fixes.append( PiecewiseLinearConstraint::Fix( _f, maxVal ) );
-	    for ( auto elem : _elements )
-		{
-		    fixes.append( PiecewiseLinearConstraint::Fix( elem, fValue ) );
-		}
-	}
+    {
+        fixes.append( PiecewiseLinearConstraint::Fix( _f, maxVal ) );
+        for ( auto elem : _elements )
+        {
+            fixes.append( PiecewiseLinearConstraint::Fix( elem, fValue ) );
+        }
+    }
     else if ( FloatUtils::lt( fValue, maxVal ) )
-	{
-	    fixes.append( PiecewiseLinearConstraint::Fix( _f, maxVal ) );
+    {
+        fixes.append( PiecewiseLinearConstraint::Fix( _f, maxVal ) );
 
```

Comment:
A few lines below, where we count the number of elements greater than f, we also need to take the eliminated variable into account.